API.md

```diff
@@ -102,18 +105,43 @@ toki.on('error', (error)=>{
 
 #### config.changed
 
-__toki__ subscribes to events triggered by __toki-config__ if the underlaying configuration mechanism detects a configuration change. __toki__ will bubble up the event for the __toki__ instantiatior to act accordingly in response to the event. 
+__toki__ subscribes to events triggered by __toki-config__ if the underlaying configuration mechanism detects a configuration change. __toki__ will bubble up the event for the __toki__ instantiatior to act accordingly in response to the event.
 
 ```Javascript
-//intantiate toki
+//instantiate toki
 const Toki = require('toki');
 const toki = new Toki({
     router : routerInstance
 });
 
 //wait for toki to be ready
 toki.on('config.changed', ()=>{
-    //toki is open for buisness (aka do your thing)    
+    //toki is open for business (aka do your thing)    
 });
 
 ```
+
+### Contracts
+_Fulfilled by the Bridge_
+
+These contracts supply a minimum. Additional properties or methods may exist, but should not be used since requests and responses provided by the bridge are often decorated versions of underlying requests or responses.
+
+#### Request
+
+Request is a decorated version of the Node http-server request object. It will always have the following:
+
++ `request.query` - a parsed query object
++ `request.params` - an object of any params from passed paths
++ `request.path` - the current path
++ `request.method` - the method which called this request
++ `request.headers` - an object containing all headers
+
+
+#### Response
```

Comment:
My understanding from our initial talks about `toki-method-http` and `toki-method-rabbit` is that the method should define the shape of its result (via templated config) and do 2 things:
  1. pass that response back to toki, which will add it to the runtime context under the action's `name` property. future potential actions now have access to this mapped result.
  2. modify the "main" response object without sending it back to the client. This way toki can build up the main response by running it through all of the methods, then at the end send the total hydrated response back to the requesting client.